src/main/scala/com/databricks/labs/overwatch/pipeline/Initializer.scala

```diff
@@ -414,18 +426,24 @@ object Initializer extends SparkSessionWrapper {
    */
   def apply(args: Array[String], debugFlag: Boolean = false): Workspace = {
 
-    logger.log(Level.INFO, "Initializing Config")
-    val config = new Config()
-    val orgId = if (dbutils.notebook.getContext.tags("orgId") == "0") {
-      dbutils.notebook.getContext.apiUrl.get.split("\\.")(0).split("/").last
-    } else dbutils.notebook.getContext.tags("orgId")
-    config.setOrganizationId(orgId)
-    config.registerInitialSparkConf(spark.conf.getAll)
-    config.setInitialShuffleParts(spark.conf.get("spark.sql.shuffle.partitions").toInt)
-    if (debugFlag) {
-      envInit("DEBUG")
-      config.setDebugFlag(debugFlag)
-    }
+    val config = initConfigState(debugFlag)
+
+    logger.log(Level.INFO, "Initializing Environment")
+    val initializer = new Initializer(config)
+    val database = initializer
+      .validateAndRegisterArgs(args)
+      .initializeDatabase()
+
+    logger.log(Level.INFO, "Initializing Workspace")
+    val workspace = Workspace(database, config)
+
+
+    workspace
+  }
+
+  private[overwatch] def apply(args: Array[String], debugFlag: Boolean, isSnap: Boolean): Workspace = {
+
+    val config = initConfigState(debugFlag)
```

Comment:
all changes in this file were to enable dbConnect. They were pulled from the Kitana pipeline work in 0.4.2